docs/requirements.txt

```diff
@@ -7,3 +10,11 @@ autodoc_pydantic==1.8.0
 sphinx-click==4.4.0
 sphinx_design==0.3.0
 sphinx-autobuild==2021.3.14
+
+# NOTE: Both the Alibi-Detect and Huggingface runtimes show a generated
+# reference of the available settings - therefore, they need to be
+# available on the environment.
+./runtimes/alibi-detect
+./runtimes/alibi-detect/requirements/dev.txt
```

Comment:
@adriangonz I think this line might need to be `-r ./runtimes/alibi-detect/requirements/dev.txt`?

Reply:
Spot on - I'll address that!